modules/polaris/src/main/ts/ephox/polaris/words/Words.ts

```diff
@@ -70,7 +70,7 @@ const findWordsWithIndices = <T>(chars: Word<T>, sChars: string[], characterMap:
         }
 
         // If the word is an abbreviation, include the next character if it's a period.
-        if (sChars[i + 1] === '.' && /^([a-zA-Z]\.)+$/.test(str + '.')) {
+        if (sChars[i + 1] === '.' && /^(([a-zA-Z]{1,2})\.)+$/.test(str + '.')) {
```

Comment:
Is there any reason to not go for `+` in case there are 3?

Reply:
Is that casting it to wide though? Like `SomethingSomethingStomething.` would be a match of that not just short abbreviations.